flow/scripts/floorplan.tcl

```diff
@@ -87,6 +87,38 @@ remove_buffers
 
 source $::env(SCRIPTS_DIR)/report_metrics.tcl
 report_metrics "floorplan final"
+set num_instances [llength [get_cells -hier *]]
+puts "number instances before restructure is $num_instances"
+puts "Design Area before restructure"
+report_design_area
+
+if { [info exist ::env(SYNTH_HIER_AREA_RECOVER)] && $::env(SYNTH_HIER_AREA_RECOVER) == 1 } {
+
```

Comment:
Would it be possible to use a different tag/variable here? It's possible one would not want to run the parts from synth.tcl, but would want to invoke it in OpenROAD.

Reply:
Ok, I can add another variable for this section.